src/firebase.js

```diff
@@ -123,40 +111,27 @@ export const fetchToken = async () => {
 
 export const onMessageListener = () =>
 	new Promise((resolve) => {
-		if (supported) {
+		if (isSupported()) {
```

Comment:
Here too :slightly_smiling_face:
```suggestion
		if (await isSupported()) {
```

Reply:
Right, I missed that!